### PR TITLE
ARM builds - initial commit to microbase 

### PR DESCRIPTION
**Pull request by niranjan92**
https://github.com/dry-dock-aarch64/microbase/issues/1

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM drydockaarch64/u16:{{%TAG%}}
```
Comment (ric03uec) [inline Dockerfile]: how is this being set?

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -e
```
Comment (ric03uec) [inline build.sh]: why is this being added if we're not hooking this repo up with the pipeline?

File: Dockerfile (hunk 1 above)
Comment (niranjan92) [reply Dockerfile]: @ric03uec setting v5.9.4 tag for now

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+FROM drydockaarch64/u16:v5.9.4
```
Comment (ric03uec) [inline Dockerfile]: this should be the last released version and not the upcoming one

Comment (niranjan92) [reply Dockerfile]: @ric03uec  updated tag to v5.9.2

**Pull request merged**